src/src/pages/capabilities/KafkaCluster/MessageContract.js

```diff
@@ -15,7 +15,7 @@ export const MessageStatus = {
   REQUESTED: "Requested",
 };
 
-function MessageHeader({
+export function MessageHeader({
   messageType,
   isOpen,
   status,
```

Comment:
Maybe we want to move this to some other file or even just MessageContracts and remove the export